src/cap-description.adoc

```diff
@@ -93,36 +93,53 @@ Therefore, it is only possible to encode a subset of all combinations.
 |==============================================================================
 
 .Encoding of architectural permissions for XLENMAX=32
-[#cap_perms_encoding32,width="60%",options=header,cols="^2,^1,^1,^1,^1,^1",align="center"]
+[#cap_perms_encoding32,width="60%",options=header,cols="^2,^1,^1,^1,^1,^1,^1,^1",align="center"]
 |==============================================================================
-| Encoding | R | W | C | X | ASR
-| 0b0000   |   |   |   |   |
-| 0b0001   5+| *reserved*
-| 0b0010   |   | ✔ |   |   |
-| 0b0011   |   | ✔ | ✔ |   |
-| 0b0100   | ✔ |   |   |   |
-| 0b0101   | ✔ |   | ✔ |   |
-| 0b0110   | ✔ | ✔ |   |   |
-| 0b0111   | ✔ | ✔ | ✔ |   |
-| 0b1000   | ✔ |   |   | ✔ |
-| 0b1001   | ✔ |   | ✔ | ✔ |
-| 0b1010   | ✔ | ✔ |   | ✔ |
-| 0b1011   | ✔ | ✔ | ✔ | ✔ |
-| 0b1100   | ✔ |   |   | ✔ | ✔
-| 0b1101   | ✔ |   | ✔ | ✔ | ✔
-| 0b1110   | ✔ | ✔ |   | ✔ | ✔
-| 0b1111   | ✔ | ✔ | ✔ | ✔ | ✔
+| Encoding  | R | W | C | X | ASR | Mode | Notes
+| 0b00000   |   |   |   |   |     |      | No permissions
+| 0b00001   | ✔ |   |   |   |     |      | Data RO
+| 0b00010   | ✔ | ✔ | ✔ | ✔ |  ✔  |  Leg | Almighty (reset)
+| 0b00011   | ✔ | ✔ | ✔ | ✔ |  ✔  | Pure | Almighty (purecap)
+| 0b00100   |   | ✔ |   |   |     |      | Data WO
+| 0b00101   7+| reserved
+| 0b00110   | ✔ | ✔ |   |   |     |      | Data RW
+| 0b00111   7+| reserved
+| 0b01000   | ✔ | ✔ | ✔ | ✔ |  ✔  |  Leg | Execute + ASR
+| 0b01001   | ✔ | ✔ | ✔ | ✔ |  ✔  | Pure | Execute + ASR
```

Comment:
Is this correct? These 2 entries are described as "execute + ASR" but the other columns are the same as the almighty cap. Shouldn't this just have X and ASR being granted?

Reply:
Duplicate removed in new version.